website/source/api/auth/kerberos/index.html.md.erb

```diff
@@ -0,0 +1,408 @@
+  "service_account": "vault_svc"
+}
+```
+
+## Read Vault Kerberos
+
+This endpoint retrieves the service account for the Kerberos auth method.
+The keytab is not returned because it is sensitive information.
+
+| Method   | Path                         |
+| :------- | :--------------------------- |
+| `GET`    | `/auth/kerberos/config`      |
+
+### Sample Request
+
+```
+$ curl \
+    --header "X-Vault-Token: ..." \
+    http://127.0.0.1:8200/v1/auth/kerberos/config
+```
+
+### Sample Response
+
+```json
+{
+  "auth": null,
+  "warnings": null,
+  "wrap_info": null,
+  "data": {
+    "service_account": "vault_svc"
+  },
+  "lease_duration": 0,
+  "renewable": false,
+  "lease_id": ""
+}
+```
+
+## Configure Kerberos LDAP
+
+This endpoint configures LDAP in the Kerberos auth method.
+
+| Method   | Path                         |
+| :------- | :--------------------------- |
+| `POST`   | `/auth/kerberos/config/ldap` |
+
+### Parameters
+
+- `url` `(string: <required>)` – The LDAP server to connect to. Examples:
+  `ldap://ldap.myorg.com`, `ldaps://ldap.myorg.com:636`. Multiple URLs can be
+  specified with commas, e.g. `ldap://ldap.myorg.com,ldap://ldap2.myorg.com`;
+  these will be tried in-order.
+- `case_sensitive_names` `(bool: false)` – If set, user and group names
+  assigned to policies within the backend will be case sensitive. Otherwise,
+  names will be normalized to lower case. Case will still be preserved when
+  sending the username to the LDAP server at login time; this is only for
+  matching local user/group definitions.
+- `starttls` `(bool: false)` – If true, issues a `StartTLS` command after
+  establishing an unencrypted connection.
+- `tls_min_version` `(string: tls12)` – Minimum TLS version to use. Accepted
+  values are `tls10`, `tls11` or `tls12`.
+- `tls_max_version` `(string: tls12)` – Maximum TLS version to use. Accepted
+  values are `tls10`, `tls11` or `tls12`.
+- `insecure_tls` `(bool: false)` – If true, skips LDAP server SSL certificate
+  verification - insecure, use with caution!
+- `certificate` `(string: "")` – CA certificate to use when verifying LDAP server
+  certificate, must be x509 PEM encoded.
+- `binddn` `(string: "")` – Distinguished name of object to bind when performing
+  user search.  Example: `cn=vault,ou=Users,dc=example,dc=com`
+- `bindpass` `(string: "")` – Password to use along with `binddn` when performing
+  user search.
+- `userdn` `(string: "")` – Base DN under which to perform user search. Example:
+  `ou=Users,dc=example,dc=com`
+- `userattr` `(string: "")` – Attribute on user attribute object matching the
+  username passed when authenticating. Examples: `sAMAccountName`, `cn`, `uid`
+- `discoverdn` `(bool: false)` – Use anonymous bind to discover the bind DN of a
+  user.
+- `deny_null_bind` `(bool: true)` – This option prevents users from bypassing
+  authentication when providing an empty password.
+- `upndomain` `(string: "")` – The userPrincipalDomain used to construct the UPN
```

Comment:
I would be open to that. The reason it is the way it is is because it's a direct copy of [the LDAP auth docs](https://github.com/hashicorp/vault/blob/master/website/source/api/auth/ldap/index.html.md.erb#L64-L67). Do you have an opinion on what's more important, being consistent or being more readable?

I honestly am flexible either way.

Reply:
I think putting it in italics just to make it stand out is fine.

Reply:
Added in https://github.com/hashicorp/vault/pull/7993/commits/56a9cec95b8869eea95cd56233e37d8848885a41 and I made sure to take a look to see it's coming out italicized.